react-app-rewired/index.ts

```diff
@@ -346,9 +346,15 @@ const reactAppRewireConfig = {
             },
             experiments: {
               topLevelAwait: true,
+              asyncWebAssembly: true,
             },
           }
-        : {},
+        : {
+            experiments: {
+              topLevelAwait: true,
+              asyncWebAssembly: true,
+            },
+          },
```

Comment:
Also related to wasm support, brings `asyncWebAssembly` in both prod and dev (which had no experiments) as well as missing `asyncWebAssembly` in prod env